understanding/21/status-messages.html

```diff
@@ -163,8 +160,8 @@ <h3>Advisory</h3>
 		<li>Using aria-live regions to support <a href="https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html">1.4.13 Content on Hover or Focus</a> (future link)</li>
```

Comment:
I need a reminder why this 1.4.13 technique that doesn't exist yet is important for 4.1.3.

Reply:
Because by having the content that is exposed by focus read out as a live region, the user does not need to do any action to hear the info, such as putting their focus on it. In some cases the exposed content may qualify as status messages, and live region is a nice technique. But since not all content exposed with hover/focus will be a status message, it is advisory.